tools/roslaunch/src/roslaunch/rlutil.py

```diff
@@ -207,6 +207,10 @@ def check_roslaunch(f):
         file_deps = {}
     for pkg, miss in missing.items():
         if miss:
+            print("Missing package dependencies: %s/package.xml: %s"%(pkg, ', '.join(miss)))
```

Comment:
I think missing package dependencies should be fixed, the point is upstream package dependencies can not be fixed by downstream  maintainers, so test should not be fail. But printing this is useful for downstream maintainers to notify upstream maintainers.